fetch.bs

```diff
@@ -1734,8 +1734,13 @@ not always relevant and might require different behavior.
 <hr>
 
 <p>A <a for=/>request</a> has an associated
-<dfn export for=request id=concept-request-priority>priority</dfn> (null or a
-user-agent-defined object). Unless otherwise stated it is null.
+<dfn export for=request id=concept-request-priority>priority</dfn>, which is
+"<code>high</code>", "<code>low</code>" or "<code>auto</code>". Unless stated otherwise it is
+"<code>auto</code>".
+
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-internal-priority>internal priority</dfn> (null or a
+<a>implementation-defined</a> object). Unless otherwise stated it is null.
```

Comment:
```suggestion
<p>A <a for=/>request</a> has an associated <dfn export for=request>internal priority</dfn> (null or
an <a>implementation-defined</a> object). Unless otherwise stated it is null.
```